_resources/port1.0/group/python-1.0.tcl

```diff
@@ -447,3 +495,20 @@ pre-livecheck {
         livecheck.type  none
     }
 }
+
+
+pre-test {
+    # disable test phase for py-<portname> stub ports
+    if {[string match py-* [option name]] && ${name} eq ${subport}} {
+          test.run      no
```

Comment:
This would perhaps be better placed in `python_set_versions`?

Reply:
Yes,  just added `test {}` there to just remove the whole phase